doc/partitioned-db.md

```diff
@@ -0,0 +1,200 @@
+# Partitioned Database
+
+When creating a Cloudant database, you can configure it as non-partitioned or partitioned.
+A partitioned database offers significant performance and cost advantages. It requires you to specify a logical partitioning of your data.
+
+To get familiar with this feature, you can read the following references as the background of our new epic:
+
+- The Cloudant official document for database partitioning: https://cloud.ibm.com/docs/services/Cloudant/guides?topic=cloudant-database-partitioning
+- The related APIs' document from our driver module `nodejs-cloudant`: https://github.com/cloudant/nodejs-cloudant#partitioned-databases
+
+_To use the following partition features, please make sure the Cloudant database that you connects to is created as partitioned._
+
+_And also make sure you are above `loopback-connector-cloudant@2.4.0` to support partition features, and if you have `@cloudant/cloudant` installed, please upgrade it to version 4.x._
+
+Now let's start with a user scenario to see how you can:
+- add partitioned index
+- create document in Partitioned Database
+- define partitioned field
+- perform partitioned find
+
+Suppose you have a Product model, with properties `id`, `name`, `tag`, `city`, and specify `city` as your partition key. You can define your model as
+
+```js
+Product = db.define('Product', {
+  name: {type: String},
+  tag: {type: String},
+  // specify `city` as the partitionKey
+  city: {type: String, isPartitionKey: true},
+  }, {
+    // include `id` field when creating a new product
+    forceId: false,
+    indexes: {
+      // create a partitioned index for frequently queried
+      // fields like `name`
+      'product_name_index': {
+        partitioned: true,
+        keys: {
+            name: 1
+        },
+      },
+    }
+  });
+```
+
+When creating a new product, you can generate its `id` field as `${city_name}: ${uuid()}`, and perform partitioned query by placing `city` in the query like: 
```

Comment:
Suggest to replace `like` with `as follows`

Reply:
changed everywhere